CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 Changes by Version
 ==================
 
+0.46.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.46.0
+* add autoscale option to enable support for Horizontal Pod Autoscaling ([#746](https://github.com/open-telemetry/opentelemetry-operator/pull/746), [@binjip978](https://github.com/binjip978))
+* chore(nodejs): bump auto-instrumentations ([#763](https://github.com/open-telemetry/opentelemetry-operator/pull/763), [@mat-rumian](https://github.com/mat-rumian))
+* Fix publishing main docker tag for operator image ([#758](https://github.com/open-telemetry/opentelemetry-operator/pull/758), [@pavolloffay](https://github.com/pavolloffay))
```

Comment:
@yuriolisa we shouldn't be adding in CI-related changes or any changes that don't make sense to end users in `CHANGELOG` as this is more focused on changes that users should be aware of from the otel-operator side. 

cc @jpkrohling @pavolloffay

Reply:
Thank you @VineethReddy02. I've just fixed it.